backend/tracim_backend/lib/webdav/middlewares.py

```diff
@@ -274,10 +284,41 @@ def __call__(self, environ, start_response):
         environ['tracim_tm'] = tm
         environ['tracim_dbsession'] = dbsession
         environ['tracim_cfg'] = self.app_config
+        registry = get_current_registry()
+        registry.ldap_connector = None
+        if AuthType.LDAP in self.app_config.AUTH_TYPES:
+            registry = self.setup_ldap(registry, self.app_config)
+        environ['tracim_registry'] =  registry
+
         app = self._application(environ, start_response)
         dbsession.close()
         return app
 
+    def setup_ldap(self, registry: Registry, app_config: CFG):
+        use_pool = True
+        pool_size = 10 if use_pool else None
+        pool_lifetime = 3600 if use_pool else None
+        get_info = None
```

Comment:
What is these parameters ? Are they important (and could be need a parameter) or it is not ?

Reply:
done